homeTask/task2.js

```diff
@@ -0,0 +1,23 @@
+
+const avarageGrade = 100
+let studentSuccess
```

Comment:
unused variable. If you do not use it then it should be deleted

Reply:
+